test/constructed_images.jl

```diff
@@ -354,4 +354,9 @@ mutable struct TestType end
         exportimagepixels!(buf2view, wand, cs, channelorder)
         @test buf2view == Ar
     end
+
+    @testset "issue #206" begin
+        img = ImageMagick.load("images/cameraman.tif")
+        @test size(img) == (512, 512)
```

Comment:
Can you also add the PNG read/write test that I posted in #206? The PNG and TiFF seem quite different use cases.

Reply:
I thought we already had tests for that right above this? Or is that a different issue?